os/android/iahwc2.cpp

```diff
@@ -230,7 +230,9 @@ HWC2::Error IAHWC2::RegisterCallback(int32_t descriptor,
   switch (callback) {
     case HWC2::Callback::Hotplug: {
       primary_display_.RegisterHotPlugCallback(data, function);
+#ifdef NESTED_DISPLAY_SUPPORT
       nested_display_.RegisterHotPlugCallback(data, function);
```

Comment:
Sorry u might need to rebase on top of master

Reply:
what's your meaning, I have prepared the patch base on latest master